sample/src/main/java/ffmpeg/videokit/sample/MainActivity.java

```diff
@@ -71,9 +71,9 @@ public void onMediaFileSelected(String path) {
 
         final Command command = videoKit.createCommand()
                 .overwriteOutput()
-                .addInputPath(path)
+                .inputPath(path)
                 .outputPath(path + POSTFIX)
-                .addCustomCommand("-ss 1 -t 3")
+                .customCommand("-ss 1 -t 3")
```

Comment:
Maybe move "-ss 1 -t 3" to private static final value and name it somehow. Just reading it I don't know what it does.